pkg/apb/rhcc_registry.go

```diff
@@ -37,6 +38,20 @@ func (r *RHCCRegistry) Init(config RegistryConfig, log *logging.Logger) error {
 	return nil
 }
 
+// This function is used because our code expects an HTTP Url for talking to RHCC
+func (r RHCCRegistry) cleanHttpUrl(url string) string {
+	if strings.HasPrefix(url, "http://") == true {
+		return url
+	}
+
+	if strings.HasPrefix(url, "https://") == true {
+		url = strings.TrimPrefix(url, "https://")
```

Comment:
Do we have a specific reason for doing this? I think it's ill-advised to quietly swap https -> http if https was knowingly specified.